src/app/admin/email-templates/email-template-form.component.ts

```diff
@@ -79,21 +92,62 @@ export class AdminEmailTemplateFormComponent extends FormComponent<EmailTemplate
     return this._ckEditorConfig;
   }
 
+  @ViewChild(CKEditorComponent, { static: false })
+  public set ckEditor(value: CKEditorComponent) {
+    this._ckEditor = value;
+  }
+
   constructor(service: AdminEmailTemplateService,
               router: Router,
               route: ActivatedRoute,
               notificationService: NotificationService,
               breadcrumbService: BreadcrumbService,
-              protected _translateService: TranslateService) {
+              protected _translateService: TranslateService,
+              protected _cdr: ChangeDetectorRef) {
     super(service, router, route, notificationService, breadcrumbService);
   }
 
   public ngOnInit() {
     this._ckEditorConfig.language = this._translateService.currentLang;
+
+    this._onLanguageChange();
+
     super.ngOnInit();
   }
 
+  public ngOnDestroy() {
+    this._destroy$.next(true);
+    this._destroy$.complete();
+  }
+
   public addText(editorArea: CKEditorComponent, text: string) {
     editorArea.instance.insertText(`{{ ' ${text} ' }}`);
   }
+
+  protected _removeCurrentEditor(): void {
```

Comment:
What is the reason to make this method protected not private?

Reply:
I don't have any reason, just followed existing style of coding of the current component.

Reply:
In this case it is better to make it private since no component extending this component.